src/image-copy/azext_imagecopy/create_target.py

```diff
@@ -47,7 +48,8 @@ def create_target_image(location, transient_resource_group_name, source_type, so
     logger.debug("storage account key: %s", target_storage_account_key)
 
     expiry_format = "%Y-%m-%dT%H:%MZ"
-    expiry = datetime.datetime.utcnow() + datetime.timedelta(hours=1)
+    expiry = datetime.datetime.utcnow() + datetime.timedelta(seconds=time_out)
+    logger.warn("create target storage sas using timeout seconds: %d", time_out)
```

Comment:
I think this should be printed for debug purposes only.
```suggestion
    logger.debug("create target storage sas using timeout seconds: %d", time_out)
```